### PR TITLE
Changes to allow local builds

### PR DESCRIPTION
**Pull request by mthornba**
- added help, local and destroy targets to Makefile
- docs for local deployment

I made all my commits to master - let me know if this needs to be merged to a branch instead.

### CODE REVIEW

**Comment by cpanato** [comment]
I will test this changes!!
thanks @mthornba for the PR

File: README.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+Config is stored in `~/.minikube`
+
+enable ingress
+```
+minikube addons enable ingress
+```
+
+Add `helm` repos
+```
+
```
Comment (cpanato) [inline README.md]: this is intentionally left blank?

**Review by cpanato — APPROVED** [review]
I followed the steps and build a minikube from scratch and Jenkins works!

File: README.md (hunk 1 above)
Comment (mthornba) [reply README.md]: Forgot about that. I couldn't remember if I needed to add any repos or if they were there by default.

Comment (cpanato) [reply README.md]: I followed the steps and did not add anything here

Comment (mthornba) [reply README.md]: I'll add a commit to remove this line.

File: mattermost-jenkins/charts/jenkins/.helmignore [hunk 2]
```diff
@@ -1,21 +0,0 @@
-# Patterns to ignore when building packages.
```
Comment (crspeller) [inline mattermost-jenkins/charts/jenkins/.helmignore]: Can you please not delete the jenkins chart? There is actually a small modification to it. Only used for the private build though. I hope to remove the need for it soon.

**Review by crspeller — CHANGES_REQUESTED** [review]
Looks good other than my one comment.

**Comment by mthornba** [comment]
I should have done a more thorough review before deleting that chart. I've restored it by copying the files from the commit prior to the delete.

**Review by crspeller — APPROVED** [review]

**Pull request merged**